unstructured/partition/image.py

```diff
@@ -81,4 +81,5 @@ def partition_image(
         languages=languages,
         strategy=strategy,
         metadata_last_modified=metadata_last_modified,
+        **kwargs,
```

Comment:
probably worth adding to docstring where `kwargs` go? (so use can anticipate what can go in?)